recipe/meta.yaml

```
@@ -187,6 +187,46 @@ outputs:
       home: https://gcc.gnu.org/
       license: GPL-3.0-only WITH GCC-exception-3.1
 
+  - name: conda-gcc-specs_{{ cross_target_platform }}                                              # [cross_target_platform == target_platform]
+    script: install-conda-specs.sh                                                                 # [cross_target_platform == target_platform]
+    build:                                                                                         # [cross_target_platform == target_platform]
+      number: {{ build_num }}                                                                      # [cross_target_platform == target_platform]
+      detect_binary_files_with_prefix: False                                                       # [cross_target_platform == target_platform]
```

Add the following and remove all the other selectors to make it look nicer.
```suggestion
      detect_binary_files_with_prefix: False                                                       # [cross_target_platform == target_platform]
      skip: True  # [cross_target_platform == target_platform]
```